lib/types.ts

```diff
@@ -693,6 +698,11 @@ export interface AccessAPI {
     entity: 'ContentType' | ContentType | 'Asset' | 'Entry' | T
   ): Promise<boolean>
   can<T = Object>(action: ArchiveableAction, entity: 'Asset' | 'Entry' | T): Promise<boolean>
+  can<T = Object>(
+    action: 'patch',
+    entity: 'Asset' | 'Entry' | T,
```

Comment:
Not quite sure about this generic. Shouldn't it just be 
```suggestion
    entity: 'Asset' | 'Entry' | string,
```
and removing the generic completely?

Or is this just to be consistent and we fix this with v4?

Reply:
could be `Task`

Reply:
Well then it's still not necessary to have a generic here. And my suggestion would still be fine.

Reply:
Per the conversation, lets keep the generic for consistency 😞